draft-irtf-cfrg-hpke.md

```diff
@@ -167,16 +178,6 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
 BCP14 {{!RFC2119}} {{!RFC8174}}  when, and only when, they appear in
 all capitals, as shown here.
 
-# Security Properties
```

Comment:
I might like to still have a summary up here, with a pointer down to the security considerations.  How about something like this (which also folds in the text from above):

> As a hybrid authenticated encryption algorithm, we desire security
> against (adaptive) chosen ciphertext attacks (IND-CCA2 secure). The
> HPKE variants described in this document achieve this property under
> standard assumptions about the underlying primitives {{HPKEAnalysis}}. 
> A summary of this analysis is in {{sec-considerations}}.

I would be OK deleting the section header in any case.